src/app/ApplicationTemplate.Droid/Properties/AndroidManifest.xml

```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android" package="com.nventive.internal.applicationtemplate" android:versionCode="1" android:versionName="1.0" android:installLocation="auto">
-	<uses-sdk android:minSdkVersion="26" android:targetSdkVersion="31" />
+	<uses-sdk android:minSdkVersion="18" android:targetSdkVersion="31" />
```

Comment:
Curious about the downgrade to 18?

Reply:
I don't remember doing that, good catch. But at the end of the day it doesn't matter this project will be removed.